post_processing/ci_plots.jl

```diff
@@ -597,6 +650,14 @@ function make_plots(
     vars = map_comparison(simdirs, short_names) do simdir, short_name
         return get(simdir; short_name, reduction)
     end
+    vars_spectra =
+        map_comparison(simdirs, short_names_spectra) do simdir, short_name
+            compute_spectrum(
+                slice(get(simdir; short_name, reduction), time = LAST_SNAP),
```

Comment:
```suggestion
                slice(get(simdir; short_name, reduction), time = 10days),
```
(We want spectra at day 10 instead of the last time step, would this work?)

Reply:
Yes, it will work